partcad/src/partcad/wrappers/wrapper_import_dxf.py

```
@@ -0,0 +1,51 @@
+# to use CadqUery
+
+import os
+import sys
+
+import cadquery as cq
+
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+
+def process(path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `process`

(ANN201)